docs/CONTRIBUTING.md

```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
 8. Under "Scopes", select "bot"
 9. Permissions. Enable only these permissions:
 
--   Manage Server
 -   Manage Roles
 -   Manage Channels
 -   Kick Members
 -   Ban Members
--   Manage Emojis
+-   Manage Emojis & Stickers
 -   Manage Webhooks
 -   View Channels
 -   All of the permissions under "Text Permissions"
--   No permissions under "Voice Permissions"
+-   Only permissions under "Voice Permissions" are Connect & Use Voice Activity.
 
 10. Select and copy the generated URL under "Scopes." Make sure the value
-    of permissions in the URL reads 4027055222
+    of permissions in the URL reads 536637602902.
 11. Send the URL via Discord DM to nonsense to have it added to the test
     Discord server
 
 ## Clone the repo to local
 
-1. Create a new feature branch from the dev branch
-2. Copy the .env file and name it whatever you want; add it to your local
-   .gitignore
+1. Create a new feature branch from the dev branch (`docs/<name>`, `feature/<name>`, `release/<name>`, `patch/<name>`).
```

Comment:
We opted to go with `hotfix/<name>` instead of `patch/<name>`

Reply:
updated ✅